features/dd-sdk-android-trace/src/main/java/com/datadog/trace/core/DDSpan.java

```diff
@@ -144,6 +145,7 @@ public DDSpan(
     }
 
     this.links = links == null ? new CopyOnWriteArrayList<>() : new CopyOnWriteArrayList<>(links);
+    this.log = LoggerFactory.getLogger(DDSpan.class.getSimpleName(), internalLogger);
```

Comment:
it is creating a logger instance...there are plenty of other more expensive objects created when a Span is created. I think we can leave with this one, otherwise we could inject the Tracer one instead but it will not have the same granularity as the tag name will always be `Tracer`.